src/stories/HdRange.stories.js

```diff
@@ -58,4 +58,25 @@ storiesOf('Form/HdRange', module)
         currentValue: this.value,
       };
     },
+  }))
+  .add('without steps', () => ({
+    components: { HdRange },
+    template: `
+      <div>
+        <HdRange
+          :minValue="0"
+          :maxValue="100"
+          :rangeStep="10"
```

Comment:
It has steps and the story says it doesn't 🤔

Reply:
FUUUUU! Bad description. It was actually without bullets. I made both the story and make the display bullet optional to make it behave more like the native one.